sdk/objc/api/peerconnection/RTCAudioTrack.mm

```diff
@@ -58,6 +61,10 @@ - (instancetype)initWithFactory:(RTC_OBJC_TYPE(RTCPeerConnectionFactory) *)facto
   return _source;
 }
 
+- (void *)getNativeTrack {
+  return _audioTrack.release();
```

Comment:
Don't we want .get() here?  Otherwise the _audioTrack will get nulled out and you can only call this once.

Same with the video version below.